requirements.txt

```diff
@@ -4,5 +4,5 @@ commonmark>=0.8,<0.10
 aiohttp>=3,<4
 yarl>=1,<2
 attrs>=19.1
-mautrix>=0.10.5,<0.11
+mautrix>=0.10.10,<0.11
```

Comment:
This caused a merge conflict since the dependency was already bumped to 0.11+

Reply:
Fixed now with a rebase.